src/NServiceBus.AzureStoragePersistence/SagaPersisters/Azure/SecondaryIndeces/LRUCache.cs

```diff
@@ -4,13 +4,13 @@ namespace NServiceBus.SagaPersisters.Azure.SecondaryIndeces
 
     public sealed class LRUCache<TKey, TValue>
```

Comment:
Please do. Since this is a cleanup, would make sense to get as much as possible. Thank you.